### PR TITLE
Rsem-calculate-expression

### PR DESCRIPTION
**Pull request by emmarousseau**
## Description

RSEM-calculate-expression component

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

**Comment by rcannood** [comment]
Hi @emmarousseau ! Does this PR need more work or is it ready for review? :relaxed:

**Comment by emmarousseau** [comment]
> Hi @emmarousseau ! Does this PR need more work or is it ready for review? ☺️

Ready now! Let me know if the test data is fine or too big.

File: src/rsem/rsem_calculate_expression/test_data/script.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+wget wget https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq3/reference/rsem.tar.gz
```
Comment (rcannood) [inline src/rsem/rsem_calculate_expression/test_data/script.sh:3]: Yo dawg I heard you like wget so I put some wget in your wget

File: src/rsem/rsem_calculate_expression/test_data/rsem.tar.gz
Comment (rcannood) [inline src/rsem/rsem_calculate_expression/test_data/rsem.tar.gz:1]: Unfortunately this is indeed too big :sweat_smile: 

In other components, I just store the fasta and gtf in the test data and build the index inside the unit test. Examples:

* [`star_align_reads`](https://github.com/viash-hub/biobox/blob/main/src/star/star_align_reads/test.sh#L74-L81)
* [`bd_rhapsody_sequence_analysis`](https://github.com/viash-hub/biobox/pull/96/files#diff-dc0e9d57cdd6e0f833ca94e8e8a91cd471a2e5559d37bf3fb63fb6b4acdf5743R30-R54)

This way, we don't need to store a large indexed reference. Could you get this to work here as well?

File: src/rsem/rsem_calculate_expression/help.txt
Comment (rcannood) [inline src/rsem/rsem_calculate_expression/help.txt:1]: This seems to be empty

**Review by rcannood — CHANGES_REQUESTED** [review]

**Review by rcannood — CHANGES_REQUESTED** [review]

**Review by rcannood — COMMENTED** [review]
Some minor requests!

:crossed_fingers: Should be good to merge afterwards

File: src/rsem/rsem_calculate_expression/test_data/output/ref.cnt
[the remark this answers is not in the document]
Comment (emmarousseau) [reply src/rsem/rsem_calculate_expression/test_data/output/ref.cnt:1]: yep makes sense haha

**Review by rcannood — APPROVED** [review]
LGTM!

**Pull request merged**